controllers/flinkcluster_util.go

```diff
@@ -26,11 +26,18 @@ import (
 )
 
 func getFlinkAPIBaseURL(cluster *v1beta1.FlinkCluster) string {
-	return fmt.Sprintf(
-		"http://%s.%s.svc.cluster.local:%d",
-		getJobManagerServiceName(cluster.ObjectMeta.Name),
+	url := fmt.Sprintf(
+		"http://localhost:8001/api/v1/namespaces/%s/services/%s:ui/proxy",
 		cluster.ObjectMeta.Namespace,
-		*cluster.Spec.JobManager.Ports.UI)
+		getJobManagerServiceName(cluster.ObjectMeta.Name))
+
+	return url
+
+	//return fmt.Sprintf(
```

Comment:
Why do you make this change? It seems to be causing problem in my test, the Flink API cannot get ready.

Reply:
Its my mistake. Sorry.